### ISSUE TITLE
.bin files missing from webforms staging folder causing DefaultApp.AppWinForms.exe to fail.

### ISSUE THREAD

**Issue by mthompson** [ServiceStackApps/ReactDesktopApps#2]
ServiceStack React Desktop app fails to run properly when compiled to self executing container.
When the app is execute, you will receive error messages in the debug.log file as follows:

[1013/105903:ERROR:renderer_main.cc(200)] Running without renderer sandbox
[1013/112107:FATAL:isolate_holder.cc(70)] Couldn't mmap v8 natives data file

I've outline the issue and proposed a solution which can be found here: 
http://atonix.clarify-it.com/d/edmlz7

Please advise if my solution is viable.

**Comment by Layoric** [comment]
@mthompson Thanks for posting with all the details, have you update the CefSharp dependencies by anychance?

I've tested it on Windows 7 and Windows 8.1 and both are successful with CefSharp 41.0.

I've noticed that for CefSharp 43.0 they have added those two files in `packages/cef.redist.{platform}/CEF/{platform/` directory.

![image](https://cloud.githubusercontent.com/assets/234642/10451521/6ffa134c-71ec-11e5-941b-03209395cd39.png)

Your fix is sensible though given they are required to run and I'll apply the same when upgrading the template to the latest CefSharp version. Thanks for posting the issue with all the details! :+1:
